staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/options/options.go

```diff
@@ -41,6 +42,7 @@ const defaultEtcdPathPrefix = "/registry/apiextensions.kubernetes.io"
 
 // CustomResourceDefinitionsServerOptions describes the runtime options of an apiextensions-apiserver.
 type CustomResourceDefinitionsServerOptions struct {
+	ServerRunOptions   *options.ServerRunOptions
```

Comment:
these changes look fine. Waiting for image discussion to settle.

Reply:
thanks for your update. :)